### PR TITLE
Fix unbounded variable issues and benchmark data saving

### PR DESCRIPTION
**Pull request by frdeso**

### CODE REVIEW

File: jobs/lttng-baremetal-tests.yaml [hunk 1]
```diff
@@ -25,6 +25,9 @@
           - text:
               credential-id: jenkins_lava_key
               variable: LAVA_JENKINS_TOKEN
+          - file:
+              credential-id: lava_jenkins_storage_frdeso
```
Comment (PSRCode) [inline jobs/lttng-baremetal-tests.yaml]: I would suggest renaming the credential id to lava_jenkins_storage

File: scripts/lttng-baremetal-tests/generate-properties-master.sh [hunk 2]
```diff
@@ -50,9 +50,16 @@ echo "LTTNG_TOOLS_COMMIT_ID=$LTTNG_TOOLS_COMMIT_ID" >> properties.txt
 
 BASE_STORAGE_FOLDER="/storage/jenkins-lava/baremetal-tests"
 
+echo "BASE_STORAGE_FOLDER=$BASE_STORAGE_FOLDER" >> properties.txt
+echo "STORAGE_HOST=storage01.internal.efficios.com" >> properties.txt
```
Comment (mjeanson) [inline scripts/lttng-baremetal-tests/generate-properties-master.sh]: Use storage.internal.efficios.com, the number could change.

File: jobs/lttng-baremetal-tests.yaml (hunk 1 above)
Comment (mjeanson) [inline jobs/lttng-baremetal-tests.yaml]: Same.

**Comment by frdeso** [comment]
Added two commits applying those changes even outside of the two initial patches.

**Pull request merged**